examples/tst_pkg_huge.pks

```diff
@@ -305,6 +305,8 @@ create or replace package tst_pkg_huge is
     -- %test(Test number 100)
     procedure test100;
     
+$IF $$tst_slim $THEN
```

Comment:
Definitely worth a simple unit test: "Only execute the code that was not disabled by conditional compilation"